pkg/v1/sdk/features/runner/runner.go

```diff
@@ -0,0 +1,155 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package runner
+
+import (
+	"context"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	k8sconfig "sigs.k8s.io/controller-runtime/pkg/client/config"
+
+	configv1alpha1 "github.com/vmware-tanzu/tanzu-framework/apis/config/v1alpha1"
+)
+
+// FeatureGateRunner defines methods to interact with FeatureGate resources
+type FeatureGateRunner struct {
```

Comment:
Rename to `FeatureClient` and update godoc to say it interacts with both `FeatureGate` and `Feature` resources.